assets/js/helpers/boot.js

```diff
@@ -14,6 +14,7 @@
     bootRstudioNavigator();
     bootAsyncLoader();
     bootPackage();
+    bootTrending();
     window.trendingPackagesLastWeek();
```

Comment:
can you move the 2 next calls to bootTrending ? Ideally the boot function only contains other sub-boot functions call